app/Http/Controllers/api/CollaboratorsController.php

```diff
@@ -15,6 +15,8 @@ public function index($area)
             return $this->collaboratorItaWiki();
         } elseif ($area === 'challenges') {
             return $this->collaboratorItaChallenges();
+        } elseif ($area === 'profiles') {
+            return $this->collaboratorItaProfiles();
         }
 
         return response()->json([
```

Comment:
I've found another way to approach this:
```suggestion
$projects = ['landing', 'wiki', 'challenges', 'profiles'];

if (!in_array($area, $projects)) {
    return response()->json([
        'message' => 'this area is invalid',
    ], 404);
} else {
    $methodName = 'collaboratorIta' . ucfirst($area);
    return $this->{$methodName}();
}```
I can't add the suggestion at L10. IDK way.